lib/compiler/webpack-compiler.ts

```diff
@@ -140,6 +140,9 @@ export class WebpackCompiler {
         return process.exit(1);
       }
       console.log(statsOutput);
+      if (!watchMode && !watch) {
+        compiler.close(() => {});
```

Comment:
Why can't we move this up - above the process.exit()?

Reply:
or just leave it here at line 143

Reply:
```
      if (!err && !stats!.hasErrors()) {
        if (!onSuccess) {
          assetsManager.closeWatchers();
        } else {
          onSuccess();
        }
      } else if (!watchMode && !watch) {
        console.log(statsOutput);
        return process.exit(1);
      }
      console.log(statsOutput);
      if (!watchMode && !watch) {
        compiler.close(() => {});
      }
```

or 

```
      if (!err && !stats!.hasErrors()) {
        if (!onSuccess) {
          assetsManager.closeWatchers();
        } else {
          onSuccess();
        }
        if (!watchMode && !watch) {
          compiler.close(() => {});
        }
      } else if (!watchMode && !watch) {
        console.log(statsOutput);
        return process.exit(1);
      }
```

Reply:
@kamilmysliwiec Would you check if you can merge this now?